Load vcs instead of package to ensure composer.json file readable

wzul opened the pull request:
Alternative. Redeclare everything declared by composer.json

```
{
    "repositories": [
      {
        "type": "package",
        "package": {
            "name": "chip/chip-sdk-php",
            "version": "v1.0.0",
            "source": {
                "url": "git@github.com:CHIPAsia/chip-php-sdk.git",
                "type": "git",
                "reference": "v1.0.0"
            },
            "autoload": {
                "classmap": [
                    "lib/"
                ]
            },
            "require": {
              "guzzlehttp/guzzle": "^7.0",
              "netresearch/jsonmapper": "^4.0"
            }
        }
      }
    ],
    "require": {
      "chip/chip-sdk-php": "^1.0"
    }
}
```

wzul committed:
Load vcs instead of package to ensure composer.json file readable

Alternative. Redeclare everything declared by composer.json

```
{
    "repositories": [
      {
        "type": "package",
        "package": {
            "name": "chip/chip-sdk-php",
            "version": "v1.0.0",
            "source": {
                "url": "git@github.com:CHIPAsia/chip-php-sdk.git",
                "type": "git",
                "reference": "v1.0.0"
            },
            "autoload": {
                "classmap": [
                    "lib/"
                ]
            },
            "require": {
              "guzzlehttp/guzzle": "^7.0",
              "netresearch/jsonmapper": "^4.0"
            }
        }
      }
    ],
    "require": {
      "chip/chip-sdk-php": "^1.0"
    }
}
```

mamadcloud commented:
can we change this part as well?
https://github.com/CHIPAsia/chip-php-sdk/blob/main/examples/composer.json to make sure it sync and works between the example and readme file?

wzul commented:
> can we change this part as well? https://github.com/CHIPAsia/chip-php-sdk/blob/main/examples/composer.json to make sure it sync and works between the example and readme file?

Sure. Will update the pr

wzul committed:
Update repositories to load vcs instead of package

Removed require guzzle and jsonmapper requirement as it defined by the package.
